system/lib/wasmfs/file.h

```diff
@@ -69,7 +83,13 @@ class File : public std::enable_shared_from_this<File> {
   std::mutex mutex;
 
 private:
-  // TODO: Add other File properties later.
+  size_t usedBytes = 0;
+
+  uint32_t mode; // r/w/x modes
+
+  time_t ctime; // Time when the inode was last modified
+  time_t mtime; // Time when the content was last modified
+  time_t atime; // Time when the content was last accessed
```

Comment:
How about using more verbose names, like `createTime, modifyTime, accessTime`?

Reply:
I am ok with that, I was just trying to match the existing POSIX format:
```
struct stat {
    dev_t     st_dev;     /* ID of device containing file */
    ino_t     st_ino;     /* inode number */
    mode_t    st_mode;    /* protection */
    nlink_t   st_nlink;   /* number of hard links */
    uid_t     st_uid;     /* user ID of owner */
    gid_t     st_gid;     /* group ID of owner */
    dev_t     st_rdev;    /* device ID (if special file) */
    off_t     st_size;    /* total size, in bytes */
    blksize_t st_blksize; /* blocksize for file system I/O */
    blkcnt_t  st_blocks;  /* number of 512B blocks allocated */
    time_t    st_atime;   /* time of last access */
    time_t    st_mtime;   /* time of last modification */
    time_t    st_ctime;   /* time of last status change */
};
```
https://linux.die.net/man/2/fstat

Reply:
I think its fine to use these short names since the match the existing known concepts.